### ISSUE TITLE
[BUG] Validator Naming

### ISSUE THREAD

**Issue by LuisNaldo7** [TobiWo/eth-duties#39]
# Issue
Parsing validator names does not work correctly. 

![image](https://user-images.githubusercontent.com/36574432/232190615-435face0-4bcf-4290-b129-69379b6329f7.png)
# Env
Version: v0.3.0
Platform: Docker

# Reproduce
```
docker run \
  --name mainnet-eth-duties \
  --network=host \
  tobiwo/eth-duties:v0.3.0 \
  --beacon-node "http://127.0.0.1:5052" \
  --validators "123456;Val_1 500000;Val_2"
```

**Comment by TobiWo** [comment]
Thanks @LuisNaldo7 . This is because you need to wrap a validator index/pubkey with it's alias within quotes. So the correct syntax would be:

```
docker run \
  --name mainnet-eth-duties \
  --network=host \
  tobiwo/eth-duties:v0.3.0 \
  --beacon-node "http://127.0.0.1:5052" \
  --validators "123456;Val_1" "500000;Val_2"
```

But I don't know whether this will work with the current `argparse` change I introduced in v0.3.0. I'm pretty sure this will be fixed while working on issue #38 . I don't have an open PR for that issue since I'm off for a couple of days. However, the fix will come within the next 2 weeks.

**Comment by LuisNaldo7** [comment]
Thanks for the hint. Wrapping both args solved the problem. Maybe we should update the docs. Not to say anything is wrong here, just to make sure noone else will fall for the same.
